core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt

```diff
@@ -285,24 +284,23 @@ fun DropdownTextField(
 
     ExposedDropdownMenuBox(
         expanded = expanded,
-        onExpandedChange = { expanded = !expanded },
+        onExpandedChange = {
+            expanded = !expanded
+        },
         modifier = modifier
     ) {
         OutlinedTextField(
             modifier = Modifier
                 .fillMaxWidth()
                 .menuAnchor(),
             value = selectedOptionText,
-            onValueChange = {},
+            onValueChange = { },
```

Comment:
Let's put and "TODO" to know in the future what we are going to do

Reply:
Please add a text regarding what is going to do inside the `TODO()`

Reply:
Sure, added it!